documentation/SRS.md

```diff
@@ -0,0 +1,109 @@
+# Human resource management web application
+
+Application should provide:
+
+-Storing departments and employees in a database;
+-Display a list of departments and the average salary for each department;
+-Display a list of employees with an indication of the employee's salary;
+-Searching employees born on a certain date or in the period between dates;
+-Updating a list of employees (adding, editing, removing);
+-Updating a list of departments (adding, editing, removing).
+
+##Departments
+### Display a list of departments
+
+The mode is designed to view a list of departments.
+
+![Main page](mockups/main_page.png)
```

Comment:
What about average salary info? Let's add this filed to the main page next to each department record.